### PR TITLE
ILS Fix @ KIAD

### PR DESCRIPTION
**Pull request by williamqarmstrong**
FIxed ILS at KIAD

### CODE REVIEW

File: Glideslope.json [hunk 1]
```diff
@@ -15759,93 +15759,93 @@
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 1.28, 
+    "bearing":  0.65, 
     "airportICAO": "KIAD", 
-    "elevation": 286, 
+    "elevation": 312.0,
```
Comment (dushyants19) [inline Glideslope.json]: try elevation in integers

Comment (ItsW1ll) [inline Glideslope.json]: remove .0

File: Glideslope.json [hunk 2]
```diff
@@ -15759,93 +15759,93 @@
...
+    "latitude": 38.93906734, 
     "identifier": "IOSZ"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 1.34, 
+    "bearing":  0.58, 
     "airportICAO": "KIAD", 
-    "elevation": 313, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline Glideslope.json]: remove .0

File: Glideslope.json [hunk 3]
```diff
@@ -15759,93 +15759,93 @@
...
+    "latitude": 38.92375485, 
     "identifier": "IIAD"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 111.48, 
+    "bearing": 110.74, 
     "airportICAO": "KIAD", 
-    "elevation": 308, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline Glideslope.json]: remove .0

File: Glideslope.json [hunk 4]
```diff
@@ -15759,93 +15759,93 @@
...
+    "bearing": 110.74, 
     "airportICAO": "KIAD", 
-    "elevation": 308, 
+    "elevation": 312.0, 
     "name": "GS", 
-    "longitude": -77.487647, 
+    "longitude": -77.49044130, 
     "associatedRunwayNumber": "12", 
     "frequency": 109300, 
     "glideslope": 3.0011074,
```
Comment (ItsW1ll) [inline Glideslope.json]: glideslope needs adding

File: Glideslope.json [hunk 5]
```diff
@@ -15759,93 +15759,93 @@
...
+    "latitude": 38.94376916, 
     "identifier": "IAJU"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 181.28, 
+    "bearing": 180.65, 
     "airportICAO": "KIAD", 
-    "elevation": 271, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline Glideslope.json]: remove .0

File: Glideslope.json [hunk 6]
```diff
@@ -15759,93 +15759,93 @@
...
+    "latitude": 38.97064707, 
     "identifier": "IDLX"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 181.34, 
+    "bearing": 180.67, 
     "airportICAO": "KIAD", 
-    "elevation": 297, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline Glideslope.json]: remove .0

File: Glideslope.json [hunk 7]
```diff
@@ -15759,93 +15759,93 @@
...
+    "latitude": 38.95532715, 
     "identifier": "ISGC"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 181.152, 
+    "bearing": 180.58, 
     "airportICAO": "KIAD", 
-    "elevation": 313, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline Glideslope.json]: remove .0

File: Glideslope.json [hunk 8]
```diff
@@ -15759,93 +15759,93 @@
...
+    "latitude": 38.97066700, 
     "identifier": "IISU"
   }, 
   {
     "receptionRange": 10, 
-    "bearing": 1.152, 
+    "bearing":  0.58, 
     "airportICAO": "KIAD", 
-    "elevation": 313, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline Glideslope.json]: remove .0

File: Glideslope.json [hunk 9]
```diff
@@ -15759,93 +15759,93 @@
...
   {
     "receptionRange": 10, 
-    "bearing": 1.152, 
+    "bearing":  0.58, 
     "airportICAO": "KIAD", 
-    "elevation": 313, 
+    "elevation": 312.0, 
     "name": "GS", 
-    "longitude": -77.47568958, 
+    "longitude": -077.47466700,
```
Comment (ItsW1ll) [inline Glideslope.json]: remove the 0 so it reads -77.47466700

File: LOC.json [hunk 10]
```diff
@@ -15822,92 +15822,92 @@
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 0.621, 
+    "bearing":  0.65, 
     "airportICAO": "KIAD", 
-    "elevation": 264, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline LOC.json]: remove .0

File: LOC.json [hunk 11]
```diff
@@ -15822,92 +15822,92 @@
...
     "identifier": "IOSZ", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 0.662, 
+    "bearing":  0.58, 
     "airportICAO": "KIAD", 
-    "elevation": 302, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline LOC.json]: remove .0

File: LOC.json [hunk 12]
```diff
@@ -15822,92 +15822,92 @@
...
     "identifier": "IIAD", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 110.724, 
+    "bearing": 110.74, 
     "airportICAO": "KIAD", 
-    "elevation": 281, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline LOC.json]: remove .0

File: LOC.json [hunk 13]
```diff
@@ -15822,92 +15822,92 @@
...
     "identifier": "IAJU", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 180.666, 
+    "bearing": 180.65, 
     "airportICAO": "KIAD", 
-    "elevation": 285, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline LOC.json]: remove .0

File: LOC.json [hunk 14]
```diff
@@ -15822,92 +15822,92 @@
...
+    "latitude": 38.97064707, 
     "identifier": "IDLX", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
     "bearing": 180.67, 
     "airportICAO": "KIAD", 
-    "elevation": 315, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline LOC.json]: remove .0

File: LOC.json [hunk 15]
```diff
@@ -15822,92 +15822,92 @@
...
     "identifier": "ISGC", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 180.576, 
+    "bearing": 180.58, 
     "airportICAO": "KIAD", 
-    "elevation": 313, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline LOC.json]: remove .0

File: LOC.json [hunk 16]
```diff
@@ -15822,92 +15822,92 @@
...
     "identifier": "IISU", 
     "type": 4
   }, 
   {
     "receptionRange": 18, 
-    "bearing": 0.576, 
+    "bearing":  0.58, 
     "airportICAO": "KIAD", 
-    "elevation": 313, 
+    "elevation": 312.0,
```
Comment (ItsW1ll) [inline LOC.json]: remove .0

File: LOC.json [hunk 17]
```diff
@@ -15822,92 +15822,92 @@
...
   {
     "receptionRange": 18, 
-    "bearing": 0.576, 
+    "bearing":  0.58, 
     "airportICAO": "KIAD", 
-    "elevation": 313, 
+    "elevation": 312.0, 
     "name": "ILS-cat-I", 
-    "longitude": -77.47431532, 
+    "longitude": -077.47466700,
```
Comment (ItsW1ll) [inline LOC.json]: remove the 0 so it reads -77.47466700

**Pull request merged**